package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-config-skyscanner",
-  "version": "2.0.0",
+  "version": "3.0.0",
```

Comment:
there was a mismatch with current release version.

Reply:
Do we really want to make this a breaking change?

Reply:
the current version on npmjs is `3.0.0`. This is not the version number update for this PR.

Reply:
Nice catch 👍